changelog/dev-5772-account-overview-type-definitions

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Move AccountOverview TypeScript namespace to shared location
```

Comment:
Do you think this changelog entry is required. I'm not sure, since it is a dev-only issue (moving files around).

Reply:
This gets compiled down to effectively no change to the user right? If that's the case, yeah, I wouldn't think we need a changelog for that.

Reply:
Yep, no change to the user plus no actual difference to the JS code once compiled.

Reply:
I've made this a "comment" changelog entry in 43ff69b, noting that it is not a user-facing change.